content/en/docs/ambient/usage/add-workloads/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Add workloads to the mesh
+description: Understand how to add workloads to an ambient mesh.
+weight: 1
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+In most cases, a cluster administrator will deploy the Istio mesh infrastructure. Once Istio is successfully deployed in ambient mode, it will be transparently available to applications deployed by all users in namespaces that have been configured to use it.
+
+## Enabling ambient for an application
+
+To add an application to the ambient mesh, add the label `istio.io/dataplane-mode=ambient`. You can apply this label to a namespace or to an individual pod.
+
+Ambient mode can be seamlessly enabled (or disabled) completely transparently as far as the application pods are concerned. Unlike when operating in {{< gloss >}}sidecar{{< /gloss >}} mode, there is no need to restart applications to add them to the mesh, and they will not show as having an extra container deployed in their pod.
+
+## Communicating between pods in different modes
+
+There are multiple options for interoperability between ambient pods and non-ambient endpoints (including Kubernetes application pods, Istio gateways or Kubernetes Gateway API instances). This interoperability provides multiple options for seamlessly integrating ambient and non-ambient workloads within the same Istio mesh, allowing for phased introduction of ambient capability as best suits the needs of your mesh deployment and operation.
+
+### Pods outside the mesh
+
+You may have namespaces which are not part of the mesh at all, in either sidecar or ambient mode. In this case, the non-mesh pods initiate traffic directly to the destination pods without going through the source node's ztunnel, while the destination pod's ztunnel enforces any L4 policy to control whether traffic should be allowed or denied.
+
+For example, setting a `PeerAuthentication` policy with mTLS mode set to `STRICT`, in the ambient namespace will cause traffic from outside the mesh to be denied.
+
+### Pods inside the mesh in sidecar mode
+
+Istio supports East-West interoperability between a pod using a sidecar proxy and an ambient pod within the same mesh. The sidecar proxy knows to use the HBONE protocol since the destination has been discovered to be an HBONE destination.
```

Comment:
```suggestion
Istio supports East-West interoperability between a pod using sidecar data plane and a pod using the ambient data plane within the same mesh. The sidecar data plane knows to use the HBONE protocol since the destination has been discovered to be an HBONE destination.
```